src/vs/workbench/browser/parts/editor/media/tabstitle.css

```diff
@@ -3,16 +3,49 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+.monaco-editor.vs .scroll-decoration {
+	box-shadow: #fff 0 6px 6px -6px inset; /* TODO: use a CSS custom property of the theme's `.monaco-editor-background` color here to blend better */
+}
+
+.monaco-editor.vs-dark .scroll-decoration {
+	box-shadow: #1E1E1E 0 6px 6px -6px inset; /* TODO: use a CSS custom property of the theme's `.monaco-editor-background` color here to blend better */
+}
```

Comment:
@bpasero are these the scroll styles I changed that you mentioned in the PR? I changed these so that they would match the editor background color so that it creates a fake gradual mask over text when you've scrolled.  That way the active tab, which has the same background color of the editor, will not appear to abruptly chop off text in the editor while also not creating a shadow (as it used to) which would make the tab appear on a different plane than the text.

Unfortunately, I had to hard-code these values, so in other themes than the just the default Dark and Light, it looks weird.  We could just hide these `display: none`, until we can get CSS custom property to give us the theme's editor base color.  I'll leave that up to you.